### ISSUE TITLE
Cannot get tooltip to open for all data points when multiple series have nearly the same x/y value

### ISSUE THREAD

**Issue by piet23** [highcharts/highcharts#21148]
#### Expected behaviour
![image](https://github.com/highcharts/highcharts/assets/24956578/705cb525-71a5-40db-9a18-445533e11248)

Given this chart, I would want to be able to see the tooltips for all data points of all series, even where they overlap ("Ascending", "Middle", and "Descending" have nearly, though not exactly, the same y value for x value 2), when hovering near them with the mouse.

With accessibility enabled, I would expect to be able to get to all data points when going up/down with the keyboard for x value 2.

#### Actual behaviour
I cannot get the tooltip for the "Middle" line to open when using the mouse.

With accessibility enabled, using the keyboard to go to the x value 2 for the "Bottom" series, when going up, only the points for "Ascending" and "Top" are reached; when going down from "Top", only the points for "Descending" and "Bottom" are reached. The only way to get to the point of the "Middle" series is to start with x value 1 and going to the right or x value 3 and going to the left.

#### Live demo with steps to reproduce
https://jsfiddle.net/piet23/ywa51942/9/

#### Product version
Highcharts 11.1.0 (not tested on older versions; also on current Highcharts 11.4.1

#### Affected browser(s)
Chrome and Firefox; not tested in other browsers

**Comment by bm64** [comment]
Hello @piet23 

Intended way to handle a situations where you want to access tooltip values for multiple points that have similar or the same position is by using `tooltip.shared` option. See the demo: https://jsfiddle.net/BlackLabel/gx64ys3p/
API reference: https://api.highcharts.com/highcharts/tooltip.shared

Let me know if it helps.

**Comment by piet23** [comment]
Hello @bm64 

Thanks for the response. Unfortunately, that does not help us. (a) We do not want to have the shared tooltips. (b) Much more severe: We have point click handlers at the data points and clicking the one of one line loads additional data for whatever that line is based on. If we cannot access one of those data points, we cannot load this extra data.

Best regards

**Comment by bm64** [comment]
Hi @piet23 

I might have misunderstood, but I checked the demo again and think I know the problem. To be on the same page, you'd want to be able to use the up and down arrow keys to navigate to each point with x position equal to 2, just like for other points that have the same x position?

**Comment by piet23** [comment]
Hi @bm64 

Correct, we want to be able to reach all data points, optimally via mouse click (which of course is difficult/impossible if the points overlap) or as a fallback via keyboard by going up/down for the points with the same x value.
